### ISSUE TITLE
Postgis result does not return country

### ISSUE THREAD

**Issue by booherbg** [marshallford/reverse-geocoder#2]
Is it possible to have Postgis result return country? Test cases:

```
POST
{
   "lat": 46.87472,
   "lng": -96.84117
}

RESPONSE:
{
    "input": {
        "lat": 46.87472,
        "lng": -96.84117
    },
    "output": "61 I- 29, Fargo, ND 58103",
    "date_retrieved": "2016-08-20T20:14:19.996Z",
    "provider": "postgis",
    "errors": []
}
```

This is how the google provider looks:

```
POST
{
   "lat": 46.874,
   "lng": -96.841
}

RESULT
{
    "input": {
        "lat": 46.874,
        "lng": -96.841
    },
    "output": "I-29, Fargo, ND 58103, USA",
    "date_retrieved": "2016-08-20T20:17:25.118Z",
    "provider": "google",
    "errors": [
        "postgis: data contains failure condition, passing over result"
    ]
}
```

**Comment by booherbg** [comment]
I just realized I can use 'provider' to see if I should expect a country code or not. I can use this for now as a workaround.

**Comment by marshallford** [comment]
See #3, solved by 2d87ea9.
